### ISSUE TITLE
natural_gradient option in fit doesn't seem to change the result

### ISSUE THREAD

**Issue by cdagnino** [stanfordmlgroup/ngboost#23]
Hi again,

Setting 

```
NGBoost(Base=default_tree_learner, Dist=Normal, Score=MLE(), natural_gradient=True)
```

vs

```
NGBoost(Base=default_tree_learner, Dist=Normal, Score=MLE(), natural_gradient=False)
```

seems to give the same results. The code below shows at least this is the case for the predictions.

I checked the source code and the line https://github.com/stanfordmlgroup/ngboost/blob/master/ngboost/ngboost.py#L21 sets the attribute `natural_gradient` but then it doesn't seem to use it anywhere else in that file.

```
from ngboost.ngboost import NGBoost
from ngboost.learners import default_tree_learner
from ngboost.scores import MLE
from ngboost.distns import Normal

from sklearn.datasets import load_boston
from sklearn.model_selection import train_test_split
from sklearn.metrics import mean_squared_error
import numpy as np

np.random.seed(seed=2334)

X, Y = load_boston(True)
X_train, X_test, Y_train, Y_test = train_test_split(X, Y, test_size=0.2)

ngb_natural = NGBoost(Base=default_tree_learner, Dist=Normal, Score=MLE(), natural_gradient=True,
                      verbose=False)

np.random.seed(seed=2334)
ngb_natural.fit(X_train, Y_train)
Y_preds_nat = ngb_natural.predict(X_test)

ngb_artificial = NGBoost(Base=default_tree_learner, Dist=Normal, Score=MLE(), natural_gradient=False,
                      verbose=False)


np.random.seed(seed=2334)
ngb_artificial.fit(X_train, Y_train)
Y_preds_art = ngb_artificial.predict(X_test)

#This one comes out True
assert np.allclose(Y_preds_nat, Y_preds_art)
```

**Comment by avati** [comment]
Ah good point. We initially had the ordinary gradient option that helped generate animations for the website. We can add it back, though we want the default to be natural gradient. Is there a particular reason you are interested in the ordinary gradient?

**Comment by alejandroschuler** [comment]
@avati it would be nice to have this option so others can perform their own ablation experiments if they choose.

**Comment by cdagnino** [comment]
Nope, no particular reason, I was just experimenting to see the difference.
